### ISSUE TITLE
About 23.03 release

### ISSUE THREAD

**Issue by chenrui333** [porjo/youtubeuploader#169]
👋 it looks like 23.03 still did not get marked as latest release. Raise this issue for some awareness. Thanks!

relates to Homebrew/homebrew-core#140768

**Comment by porjo** [comment]
Hi @chenrui333 not every tag will be a release. Is this causing a problem?

**Comment by chenrui333** [comment]
that would be a problem for homebrew packaging simply because you used the github releases before, let me know if that makes sense.

**Comment by chenrui333** [comment]
mostly just matter of consistency (we ran into lots of checksum and re-tagging with other packages before) and this release note process would help with it (also the release notes can be auto-generated now)

**Comment by porjo** [comment]
Sorry, I don't know anything about how Homebrew works. Can it be made to look at releases, rather than tags?

Anyhow, I've made a new release now so hopefully that will resolve your issue.

**Comment by ZhongRuoyu** [comment]
> Can it be made to look at releases, rather than tags?

Yes, it's possible. I've updated our livecheck strategy for `youtubeuploader` in Homebrew/homebrew-core#140768 so hopefully we won't have such issues in the future. Thanks @porjo for confirming!
